appliance/postgresql/client/client.go

```diff
@@ -1,20 +1,25 @@
 package pgmanager
 
 import (
+	"errors"
 	"fmt"
 	"net"
 	"net/http"
 	"time"
 
 	"github.com/flynn/flynn/appliance/postgresql/state"
+	"github.com/flynn/flynn/discoverd/client"
 	"github.com/flynn/flynn/pkg/httpclient"
 )
 
 type PostgresInfo struct {
-	Config   *state.PgConfig `json:"config"`
-	Running  bool            `json:"running"`
-	XLog     string          `json:"xlog,omitempty"`
-	Replicas []*Replica      `json:"replicas,omitempty"`
+	Config     *state.PgConfig     `json:"config"`
+	Running    bool                `json:"running"`
+	SyncedInst *discoverd.Instance `json:"synced_inst"`
```

Comment:
I'm not a huge fan of the `SyncedInst` field name. Is `SyncedDownstream` accurate? I don't have any great ideas.